experimental/origin-admin/src/pages/marketplace/mutations/_demoListings.js

```diff
@@ -1,8 +1,9 @@
 const HawaiiHouse = {
-  schemaId: 'http://schema.originprotocol.com/listing_v1.0.0',
+  schemaId: 'https://schema.originprotocol.com/listing_1.0.0.json',
+  dappSchemaId: "https://dapp.originprotocol.com/schemas/forRent-housing_1.0.0.json",
   listingType: 'unit',
-  category: 'schema.housing',
-  subCategory: 'schema.housing.vacationRentals',
+  category: 'schema.forRent',
+  subCategory: 'schema.forRent.housing',
```

Comment:
@franckc - in my PR here:
https://github.com/OriginProtocol/origin/pull/1026
I removed the second namespace in schema message IDs so they're all just `schema.<messageId>` because that allowed me to simplify the code quite a bit.

Reply:
Ah ok. I can make that change. If we are going that route, do we even need to keep the "schema." prefix for category and subCategory ?

Reply:
Personally, I think it makes sense to keep the `schema.` prefix because it helps to differentiate translatable "message IDs" that exist in our dictionary from normal strings - open to feedback/ideas though.

Reply:
Done. @jordajm ready for another round of review...